man/systemd.network.xml

```diff
@@ -3643,6 +3643,15 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>FirewallMark=</varname></term>
+        <listitem>
+          <para>Takes an unsigned integer in the range 1…4294967295. When specified, firewall-mark-based
```

Comment:
as above, i'd drop the "unsigned", it's redundant info given you specify the literal range anyway